src/Files.App/Views/MainPage.xaml.cs

```diff
@@ -112,15 +100,6 @@ private async Task AppRunningAsAdminPromptAsync()
 				UserSettingsService.ApplicationSettingsService.ShowRunningAsAdminPrompt = false;
 		}
 
-		// WINUI3
-		private ContentDialog SetContentDialogRoot(ContentDialog contentDialog)
```

Comment:
This is required to prevent a crash

Reply:
I can build on my end and tested all stuff, and CI even fails to build.

Reply:
The CI wouldn't catch this as `PromptForReviewAsync` is only called in certain cases.

Reply:
But this is used from nowhere.

Reply:
Should be fine in this case 👍